cmd/mirror.go

```diff
@@ -374,6 +379,20 @@ func newMirrorPublishCmd() *cobra.Command {
 	return cmd
 }
 
+func validatePlatform(goos, goarch string) error {
+	// Only support any/any, don't support linux/any, any/amd64 .etc.
+	if goos == "any" && goarch == "any" {
```

Comment:
I don't think this bool expr is correct

Reply:
It's correct. any/any is supported, any/amd64 not